### ISSUE TITLE
Issue 4: Focus indicator missing

### ISSUE THREAD

**Issue by artjynx** [TAMULib/tamu-library-components#425]
It must be possible for a person to know which element among multiple elements has the keyboard focus. A keyboard only user navigates the interactive links on the page with tab key and focus indicator should display the current focused element on the screen.
Navigation buttons missing visual focus indicator
main navigation missing visual focus indicator
Accessibility guidelines
[2.4.7 Focus Visible](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FTR%2FWCAG21%2F%23focus-visible&data=05%7C01%7Cartjynx%40library.tamu.edu%7Cabb64fa5c8244c6c447e08da64cbe4ac%7C68f381e346da47b9ba576f322b8f0da1%7C1%7C0%7C637933124123818269%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=YaJrSr0e%2BxXqPSDD5q56P72YAAuowcNqvu2%2FX51zezc%3D&reserved=0) (Level AA): Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.
Recommendation
Visit [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://nam02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.w3.org%2FWAI%2FWCAG21%2FTechniques%2Fcss%2FC15&data=05%7C01%7Cartjynx%40library.tamu.edu%7Cabb64fa5c8244c6c447e08da64cbe4ac%7C68f381e346da47b9ba576f322b8f0da1%7C1%7C0%7C637933124123974916%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=eVnuBxrAhkIiKAkSrT3YQ25u43PjCO2zY1Q6vQKcuEo%3D&reserved=0) for resolving this issue.

**Comment by jcreel** [comment]
Work on this in conjunction with #423
